### PR TITLE
zh-translation:content/zh/docs/tasks/security/authentication/authn-policy/index.md

### PR DESCRIPTION
**Pull request by mrshengzyzy**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/authentication/authn-policy/index.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ EOF
 {{< /text >}}
 
 1. The port value in the peer authentication policy is the container's port. The value the destination rule is the service's port.
-1. You can only use `portLevelMtls` if the port is bound to a service. Istio ignores it otherwise.
+2. You can only use `portLevelMtls` if the port is bound to a service. Istio ignores it otherwise.
```
Comment (ericvn) [inline content/en/docs/tasks/security/authentication/authn-policy/index.md:306]: The lines should all be numbered with `1.` and not increment each section.

**Review by ericvn — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/security/authentication/authn-policy/index.md [hunk 2]
```diff
@@ -589,19 +589,19 @@ $ curl "$INGRESS_HOST:$INGRESS_PORT/ip" -s -o /dev/null -w "%{http_code}\n"
     $ kubectl -n istio-system delete requestauthentication jwt-example
     {{< /text >}}
 
-1. Remove authorization policy:
+2. Remove authorization policy:
```
Comment (ericvn) [inline content/en/docs/tasks/security/authentication/authn-policy/index.md:592]: Same as above. All items should have `1.`

**Review by ericvn — CHANGES_REQUESTED** [review]

File: content/en/docs/tasks/security/authentication/authn-policy/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/docs/tasks/security/authentication/authn-policy/index.md:306]: It's also good that the linter has identified these as well.

Comment (mrshengzyzy) [reply content/en/docs/tasks/security/authentication/authn-policy/index.md]: thx for check，I correct the line number and indents.

File: content/en/docs/tasks/security/authentication/authn-policy/index.md [hunk 3]
```diff
@@ -585,24 +585,24 @@ $ curl "$INGRESS_HOST:$INGRESS_PORT/ip" -s -o /dev/null -w "%{http_code}\n"
 
 1. Remove authentication policy:
 
-    {{< text bash >}}
-    $ kubectl -n istio-system delete requestauthentication jwt-example
-    {{< /text >}}
+{{< text bash >}}
```
Comment (ericvn) [inline content/en/docs/tasks/security/authentication/authn-policy/index.md:588]: Removing the indents here have now caused the numbering to reset, so they all show as 1. in the deployed page (see https://deploy-preview-9419--preliminary-istio.netlify.app/latest/docs/tasks/security/authentication/authn-policy/#cleanup-part-3). The netlify job details will point to a generated website so you can view what you changes actually look like.

**Comment by ericvn** [comment]
You will have to run `make gen` to generate a new snips.sh since you edited one of the scripts on the webpage (noted by the failed gencheck job).

**Comment by ericvn** [comment]
You can also look at the `lint` details to see what is failing there. Running `make lint` locally will highlight the actual misspelled words in case it's to obvious from the logs which don't show the highlighting.

File: content/en/docs/tasks/security/authentication/authn-policy/index.md (hunk 3 above)
Comment (mrshengzyzy) [reply content/en/docs/tasks/security/authentication/authn-policy/index.md]: I rechecked the code and then use `Make Gen` and `make lint` to check it. It should be all right now.

**Comment by mrshengzyzy** [comment]
@googlebot I fixed it.

**Comment by mrshengzyzy** [comment]
@googlebot I fixed it.

**Comment by mrshengzyzy** [comment]
there is something wrong with cla，and I don't know how to resolve it. So closed and create new one.

**Comment by craigbox** [comment]
A commit was authored by "root@shengzheng0100.home.langchao.com".
I can override this if necessary, or you can edit the history to change the e-mail address of the commit.

**Pull request closed without merging**